backend/db/crud_form.py

```diff
@@ -131,7 +131,7 @@ def generate_webform_json(session: Session, id: int):
                         if d['operator'] == OperatorType.less_than_or_equal:
                             operator = "max"
                         d.update({operator: value})
-                    del d['id']
+                    del d['dependent_to']
```

Comment:
If the list came from somewhere then we modify it by using `del`, it might unknowningly cause a hard-to-detect side effects btw. You can modify them later with Pydantic.

Reply:
Noted mas @dedenbangkit. I Will updated it later.